setuptools/command/editable_wheel.py

```diff
@@ -600,7 +600,7 @@ def _simple_layout(
     layout = {pkg: find_package_path(pkg, package_dir, project_dir) for pkg in packages}
     if not layout:
         return set(package_dir) in ({}, {""})
-    parent = os.path.commonpath(starmap(_parent_path, layout.items()))
+    parent = os.path.commonpath(starmap(_parent_path, layout.items()))  # type: ignore[call-overload] # FIXME upstream
```

Comment:
Upstream fix: https://github.com/python/typeshed/pull/11310

Reply:
Maybe add the link to the comment?